ps_facetedsearch.php

```diff
@@ -90,7 +90,7 @@ public function __construct()
     {
         $this->name = 'ps_facetedsearch';
         $this->tab = 'front_office_features';
-        $this->version = '3.2.1';
+        $this->version = '3.2.2';
```

Comment:
Since it's not a regression it must be 3.3.0 :)
But please, don't update the version in the same PR, it's not related to the fix.

Reply:
oh okay.

Reply:
The fix is not related to the module, only to the Core.
The version will be updated depending on what we want to release.